addons/account/static/src/components/account_payment_field/account_payment.xml

```diff
@@ -69,33 +69,41 @@
                 <div>
                     <table>
                         <tr>
-                            <td><strong>Amount: </strong></td>
-                            <td>
+                            <td class="fw-bolder">Amount:</td>
+                            <td class="ps-1">
                                 <t t-out="props.amount_company_currency"></t>
                                 <t t-if="props.amount_foreign_currency">
                                     (<span class="fa fa-money"/> <t t-out="props.amount_foreign_currency"/>)
                                 </t>
                             </td>
                         </tr>
                         <tr>
-                            <td><strong>Memo: </strong></td>
-                            <td>
-                                <div style="width: 200px; word-wrap: break-word">
+                            <td class="fw-bolder align-top">Memo:</td>
+                            <td class="ps-1 position-relative">
+                                <div class="o_memo_content" t-ref="memoContent" t-on-mouseenter="onMemoHover" t-on-mouseleave="onMemoLeave">
+                                    <t t-out="props.ref"/>
+                                </div>
+                                <div class="o_memo_tooltip" t-ref="memoTooltip">
                                     <t t-out="props.ref"/>
                                 </div>
```

Comment:
```suggestion
                                <div class="o_memo_content" t-att-data-tooltip="props.ref">
                                    <t t-out="props.ref"/>
                                </div>
```
I think we probably should use a `tooltip` like this. It will simplify everything a lot. What do you think ?

Reply:
Yes, indeed with the recent change of specs, we can use it.